app/models/ahoy/event.rb

```diff
@@ -0,0 +1,8 @@
+class Ahoy::Event < ApplicationRecord
+  include Ahoy::QueryMethods
+
+  self.table_name = "ahoy_events"
+
+  belongs_to :visit
+  belongs_to :user, optional: true
+end
```

Comment:
`events` and `visits` are new models added when setting up the gem to track the user visits